package.json

```diff
@@ -31,36 +31,41 @@
     "pretty": "prettier --write src"
   },
   "devDependencies": {
-    "@types/chai": "^4.2.12",
-    "@types/glob": "^7.1.3",
-    "@types/jquery": "^3.5.5",
-    "@types/markdown-it": "^12",
-    "@types/mocha": "^8.0.3",
-    "@types/node": "^15",
-    "@types/pako": "^1.0.1",
-    "@types/plotly.js": "^1.50.15",
-    "@types/tmp": "^0.2.0",
-    "@typescript-eslint/eslint-plugin": "^4.5.0",
-    "@typescript-eslint/parser": "^4.5.0",
+    "@types/chai": "^4.2.21",
+    "@types/glob": "^7.1.4",
+    "@types/jquery": "^3.5.6",
+    "@types/markdown-it": "^12.0.3",
+    "@types/mocha": "^8.2.3",
+    "@types/node": "^15.14.1",
+    "@types/pako": "^1.0.2",
+    "@types/plotly.js": "^1.54.11",
+    "@types/tmp": "^0.2.1",
+    "@typescript-eslint/eslint-plugin": "^4.28.2",
+    "@typescript-eslint/parser": "^4.28.2",
     "3dmol": "^1.6.4",
     "assert": "^2.0.0",
     "bootstrap": "^4.6.0",
     "bubleify": "^2.0.0",
     "buffer": "^6.0.2",
-    "canvas": "^2.6.1",
+    "canvas": "^2.8.0",
```

Comment:
I don't think we need canvas anymore, it was used to provide a WegGL context inside node for the previous tests.